examples/with-i18n-next-intl/README.md

```diff
@@ -0,0 +1,21 @@
+# next-intl example
+
+This example uses [next-intl](https://github.com/amannn/next-intl), a minimal, but complete solution for managing internationalization in Next.js apps.
```

Comment:
Maybe include a few points on why this library versus others.

Reply:
I've copied the USP description from my README here now. Does that make sense to you? To be honest, I wouldn't want to compare too directly with other libraries, since there are always some tradeoffs. I guess ideally the user is able to make a decision based on the unique features a library offers.

This is something I always liked about Next.js, that there's no "Why we're better than Gatsby" section 😄 .

What do you think?